docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -822,6 +828,40 @@
                 </tr>
 
                 <tr>
+                    <th colspan="3" class="section" id="streams"><a href="../container/streams.html">Streams</a></th>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-system">streams.<span class="stream">stream-id</span>.samza.system</td>
+                    <td class="default"></td>
+                    <td class="description">
+                        The <span class="system">system-name</span> of the system on which this stream will be accessed.
+                        This property binds the stream to one of the systems defined with the property
+                        systems.<span class="system">system-name</span>.samza.factory. <br>
+                        If this property isn't specified, it is inherited from job.default.system.
+                    </td>
+                </tr>
+
+                <tr>
+                    <td class="property" id="streams-physical-name">streams.<span class="stream">stream-id</span>.samza.physical.name</td>
```

Comment:
- Don't we already have properties of the following format - `systems.system-name.
streams.stream-name.samza.key.serde` ? (also, offset reset default settings are another example). Here, the property name seems to be `streams.stream-name.*`.?  ie, the specification of system and stream is interchanged. I suspect that's intentional? Wondering if we can unify the two? (or are the two meant to co-exist?)

- Also, if I want to set `system` level properties, should I set all of those properties for each of the `streams` in the system?